DQM/SiPixelMonitorCluster/src/SiPixelClusterSource.cc

```diff
@@ -132,6 +132,20 @@ void SiPixelClusterSource::bookHistograms(DQMStore::IBooker & iBooker, edm::Run
     ss2.str(std::string()); ss2 << "Clusters -Z Disk" << i << ";Global X (cm);Global Y (cm)";
     meClPosDiskmz.push_back(iBooker.book2D(ss1.str(),ss2.str(),80,-20.,20.,80,-20.,20.));
   }
+
+  //Book trend cluster plots for barrel and endcap. Lumisections for offline and second for online - taken from strips
+  iBooker.setCurrentFolder(topFolderName_+"/Barrel");                                           
+  ss1.str(std::string()); ss1 << "totalNumberOfClustersProfile_siPixelClusters_Barrel";
+  ss2.str(std::string()); ss2 << "Total number of barrel clusters profile;Lumisection;";
+  meClusBarrelProf = iBooker.bookProfile(ss1.str(),ss2.str(),2400,0.,150,0,0,"");
+  meClusBarrelProf->getTH1()->SetCanExtend(TH1::kAllAxes);
+
+  iBooker.setCurrentFolder(topFolderName_+"/Endcap");                                     
+  ss1.str(std::string()); ss1 << "totalNumberOfClustersProfile_siPixelClusters_Endcap";
+  ss2.str(std::string()); ss2 << "Total number of endcap clusters profile;Lumisection;";
+  meClusEndcapProf = iBooker.bookProfile(ss1.str(),ss2.str(),2400,0.,150,0,0,"");
+  meClusEndcapProf->getTH1()->SetCanExtend(TH1::kAllAxes);
```

Comment:
@dmitrijus these kind of plots require a binning that depends on the run lenght (the x axis variable is the number of lumisections), we could also set a fixed range, but in this case the number of bins should be choosen in a way to cover "all possible" run lenghts, so it should be choosen as a big number even if the specific run turns out to be a short one. So, I would prefer to keep the "can extend option" unless it really break something. Let me know if this is a an issue, for my recollection these are the only plots to be added with this feature (and btw, the correspondent plots for SiStrip have the same settings)